.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -21,6 +21,19 @@ jobs:
       done
     displayName: Manage disk space
 
+  - script: |
```

Comment:
This is more vorbose then it needs to be, see https://github.com/conda-forge/conda-smithy/pull/1751